src/angular/src/app/pages/settings/path-pairs.component.scss

```
@@ -0,0 +1,214 @@
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    font-size: 80%;
+    font-weight: 500;
+    padding: 10px 15px;
+    background-color: var(--ss-header-bg);
+    border-bottom: 1px solid var(--ss-border);
+
+    .btn-add {
+      @extend %button;
+      padding: 2px 10px;
```

Stylelint (17.3.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)